### PR TITLE
add sql file for lgd_category

### PR DESCRIPTION
**Pull request by fikrihaykal41**
first commit for `lgd_category`

### CODE REVIEW

File: source/sql/dwh/bq_view/L1/visibility/lgd_category.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  CAST(REPLACE(JSON_EXTRACT(data, '$.totalProductBind'), '"', '') AS INT64) AS total_productbind,
+  CAST(REPLACE(JSON_EXTRACT(data, '$.isDeleted'), '"', '') AS BOOL) AS is_deleted,
+  REPLACE(JSON_EXTRACT(data, '$.createdBy'), '"', '') AS created_by,
+  CAST(REPLACE(JSON_EXTRACT(data, '$.createdAt'), '"', '') AS TIMESTAMP) AS created_at,
+  REPLACE(JSON_EXTRACT(data, '$.modifiedBy'), '"', '') AS modified_by,
+  CAST(REPLACE(JSON_EXTRACT(data, '$.modifiedAt'), '"', '') AS TIMESTAMP) AS modified_at,
+  IF(REPLACE(JSON_EXTRACT(data, '$.parentId'), '"', '') = "", NULL, REPLACE(JSON_EXTRACT(data, '$.parentId'), '"', ''))  AS parent_id,
+  data AS original_data,
+  ts AS published_timestamp
+FROM `logee-data-dev.logee_datalake_raw_development.visibility_lgd_category`
```
Comment (toileto) [inline source/sql/dwh/bq_view/L1/visibility/lgd_category.sql:15]: Jangan lupa di ganti ke `logee-data-prod.logee_datalake_raw_production` yaaa

File: source/sql/dwh/bq_view/L1/visibility/lgd_category.sql [hunk 2]
```diff
@@ -0,0 +1,16 @@
+SELECT
+  IF(REPLACE(JSON_EXTRACT(data, '$.categoryId'), '"', '') = "", NULL, REPLACE(JSON_EXTRACT(data, '$.categoryId'), '"', ''))  AS category_id,
+  REPLACE(JSON_EXTRACT(data, '$.categoryName'), '"', '') AS category_name,
+  IF(REPLACE(JSON_EXTRACT(data, '$.categoryImage'), '"', '') = "", NULL, REPLACE(JSON_EXTRACT(data, '$.categoryImage'), '"', ''))  AS category_image,
+  REPLACE(JSON_EXTRACT(data, '$.categoryType'), '"', '') AS category_type,
+  CAST(REPLACE(JSON_EXTRACT(data, '$.totalProductBind'), '"', '') AS INT64) AS total_productbind,
```
Comment (toileto) [inline source/sql/dwh/bq_view/L1/visibility/lgd_category.sql:6]: Mungkin dibikin `total_product_bind` aja

**Review by toileto — CHANGES_REQUESTED** [review]
Check comment yaaa Haykal

**Review by fikrihaykal41 — COMMENTED** [review]
oke oke

**Review by ameliaksalwa — APPROVED** [review]

**Pull request merged**